proto/one/enterprise/data/dataset_definition_v2.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+message EnterpriseDataSourceDefinition {
+    // DataSoruce name
+    string name = 1;
+
+    // Optional: A list of key value configuration options for the datasource data retrieval
+    map<string, google.protobuf.StringValue> options = 2;
+}
+
+message EnterpriseDataTableQueryParameter {
```

Comment:
What are these queryparameters used to query?  If they are going to be in a url they would always be strings right?

Reply:
I will remove the type, it was added by the JSON example I saw at the wiki page, we can always use string since the data source driver implementation can convert string to any type it needs and then apply down stream query filtering.

Reply:
EnterpriseDataTableQuery is used to persist parameters to dataset definition and used for future dataset refresh, the refresh dataset endpoint will be provided later, or our system has a scheduler job refresh a dataset upon dataset settings, detail design about that part will proceed later as well with Jacob.